### ISSUE TITLE
Using IgnorePlugin produce error with SBG

### ISSUE THREAD

**Issue by webleaf** [NativeScript/nativescript-dev-webpack#768]
**Environment**
✔ Component nativescript has 5.1.1 version and is up to date.
✔ Component tns-core-modules has 5.1.2 version and is up to date.
✔ Component tns-android has 5.1.0 version and is up to date.
✖ Component tns-ios is not installed.
 - Node.js: v10.4.0
 

**Describe the bug**

I try to exclude some large modules from some package to reduce `vendor.js` size. Looks like it somehow works, but some error are produced. One is about `sbg-bindings.txt` (this file not generated if I use `IgnorePlugin`)
```
java.io.FileNotFoundException: /path/to/project/platforms/android/build-tools/sbg-bindings.txt (No such file or directory)
        at java.io.FileInputStream.open0(Native Method)
        at java.io.FileInputStream.open(FileInputStream.java:195)
        at java.io.FileInputStream.<init>(FileInputStream.java:138)
        at java.io.FileInputStream.<init>(FileInputStream.java:93)
        at org.nativescript.staticbindinggenerator.Generator.getRows(Generator.java:148)
        at org.nativescript.staticbindinggenerator.Generator.generateBindings(Generator.java:88)
        at org.nativescript.staticbindinggenerator.Generator.writeBindings(Generator.java:66)
        at org.nativescript.staticbindinggenerator.Main.main(Main.java:47)
```

**To Reproduce**

Add `IgnorePlugin` to plugin section and exclude some module from some package:
```
new webpack.IgnorePlugin({
  resourceRegExp: /someModuleName/,
  contextRegExp: /somePackageName/
}),
```
Then additionaly I try so (add `!(/someModuleName/.test(moduleName))` condition to exclude module from chunk):
```
splitChunks: {
                cacheGroups: {
                    vendor: {
                        name: "vendor",
                        chunks: "all",
                        test: (module, chunks) => {
                            const moduleName = module.nameForCondition ? module.nameForCondition() : '';
                            return  (/[\\/]node_modules[\\/]/.test(moduleName) ||
                                     appComponents.some(comp => comp === moduleName))
                                  && !(/someModuleName/.test(moduleName));

                        },
                        enforce: true,
                    },
                }
            },
```
Error disappear, `sbg-bindings.txt` appear, but this file now contain only few strings for `someModuleName` (normally there are a lot of strings).

Also using `IgnorePlugin` with `--env.report` produce:
```
Error parsing bundle asset "/path/to/project/platforms/android/app/src/main/assets/app/vendor.js": Unexpected token (68920:27)
```

**Comment by poppahorse** [comment]
@webleaf why did you close? was this resolved? Also getting this issue, but doesn't seem to be caused by `IgnorePlugin`, and is getting thrown when I try to build the apk via a docker container on a gitlab runner (interestingly the same docker container locally works fine)

**Comment by webleaf** [comment]
@poppahorse I decided that maybe I'm using `IgnorePlugin` wrong. Because after cutting some module via `IgnorePlugin`, there are still remain `require` lines to that module (but I know that this lines will not be called in my case, thats why I've tried cut that module so boldly). But looks like SBG still keep do job based on this "broken dependencies".
So I've resolved my issue without `IgnorePlugin` by manually cutting off needless code going through all `require` calls and commenting them.
